### ISSUE TITLE
[BUG]Frequency not provided and cannot be inferred

### ISSUE THREAD

**Issue by Enderfga** [autogluon/autogluon#2128]
I'm using autogluon to predict time series data, my data is larger in size but generally very similar in structure to the example with the number of COVID-19 in each country. I have 20k items with an average time series length of about 450 and am using an RTX 2080Ti , I edited my code exactly as the [example](https://auto.gluon.ai/stable/tutorials/timeseries/forecasting-quickstart.html), only after making corresponding changes for my data, but no matter how I run it, it won't train properly, is this a video memory issue?

presets is set to best_quality
================ TimeSeriesPredictor ================
TimeSeriesPredictor.fit() called
Setting presets to: best_quality
Fitting with arguments:
{'evaluation_metric': 'MAPE',
 'hyperparameter_tune_kwargs': {'num_trials': 20,
                                'scheduler': 'local',
                                'searcher': 'random'},
 'hyperparameters': 'default_hpo',
 'prediction_length': 5,
 'target_column': 'prb',
 'time_limit': None}
Provided training data set with 8693213 rows, 20000 items. Average time series length is 434.66065.

[BUG]Frequency not provided and cannot be inferred

**Comment by shchur** [comment]
Can you please mention what version of autogluon you are using?
```python
import autogluon.timeseries as agts
print(agts.__version__)
```

The most likely reason for this error is that the dataset you're working with contains irregularly sampled time series (e.g., the time series is not recorded each day). 

If you use the latest version of AutoGluon (from the master branch in this repo, something like `0.5.3b...`), you can solve this problem by setting the `ignore_time_index` flag for the `TimeSeriesPredictor`
```python
predictor = TimeSeriesPredictor(..., ignore_time_index=True)
```

If you use the release version (e.g., `0.5.2`, `0.5.1`, `0.5.0`), you will need to manually fill the missing values in the dataset, e.g., using [forward filling](https://datascience.stackexchange.com/questions/57776/explain-forward-filling-and-backward-filling-data-filling).

**Comment by Enderfga** [comment]
I'm using 0.5.2, how do I install the master branch of the repo, I've only used the release version

**Comment by Enderfga** [comment]
Just use pip3 install --pre autogluon and I am good to go?

**Comment by shchur** [comment]
Either with `pip install --pre autogluon` or by installing from source using the official documentation https://auto.gluon.ai/stable/index.html

**Comment by Enderfga** [comment]
Thank you very much for your help, ignore_time_index=True works, but it still reports the same error

**Comment by Enderfga** [comment]
ERROR: Could not find a version that satisfies the requirement autogluon.common==0.5.3b20220913 (from autogluon-core[all,tests]) (from versions: 0.3.2b20211207, 0.3.2b20211208, 0.3.2b20211209, 0.3.2b20211210, 0.3.2b20211211, 0.3.2b20211212, 0.3.2b20211213, 0.3.2b20211214, 0.3.2b20211215, 0.3.2b20211216, 0.3.2b20211217, 0.3.2b20211218, 0.3.2b20211219, 0.3.2b20211220, 0.3.2b20211221, 0.3.2b20211222, 0.3.2b20211223, 0.3.2b20211224, 0.3.2b20211225, 0.3.2b20211226, 0.3.2b20211227, 0.3.2b20211228, 0.3.2b20211229, 0.3.2b20211230, 0.3.2b20211231, 0.3.2b20220101, 0.3.2b20220102, 0.3.2b20220103, 0.3.2b20220104, 0.3.2b20220105, 0.3.2b20220106, 0.3.2b20220107, 0.3.2b20220108, 0.3.2b20220109, 0.3.2b20220110, 0.3.2b20220111, 0.3.2b20220112, 0.3.2b20220113, 0.3.2b20220114, 0.3.2b20220115, 0.3.2b20220116, 0.3.2b20220117, 0.3.2b20220118, 0.3.2b20220119, 0.3.2b20220120, 0.3.2b20220121, 0.3.2b20220122, 0.3.2b20220123, 0.3.2b20220124, 0.3.2b20220125, 0.3.2b20220126, 0.3.2b20220127, 0.3.2b20220128, 0.3.2b20220129, 0.3.2b20220130, 0.3.2b20220131, 0.3.2b20220201, 0.3.2b20220202, 0.3.2b20220203, 0.3.2b20220204, 0.3.2b20220205, 0.3.2b20220206, 0.3.2b20220207, 0.3.2b20220208, 0.3.2b20220209, 0.3.2b20220210, 0.3.2b20220211, 0.3.2b20220212, 0.3.2b20220213, 0.3.2b20220214, 0.3.2b20220215, 0.3.2b20220216, 0.3.2b20220217, 0.3.2b20220218, 0.3.2b20220219, 0.3.2b20220220, 0.3.2b20220221, 0.3.2b20220222, 0.3.2b20220223, 0.3.2b20220224, 0.3.2b20220225, 0.3.2b20220226, 0.3.2b20220227, 0.3.2b20220228, 0.3.2b20220301, 0.3.2b20220302, 0.3.2b20220303, 0.3.2b20220304, 0.3.2b20220305, 0.3.2b20220306, 0.3.2b20220307, 0.3.2b20220308, 0.3.2b20220309, 0.4.0, 0.4.1b20220310, 0.4.1b20220311, 0.4.1b20220312, 0.4.1b20220313, 0.4.1b20220314, 0.4.1b20220315, 0.4.1b20220316, 0.4.1b20220317, 0.4.1b20220318, 0.4.1b20220319, 0.4.1b20220320, 0.4.1b20220321, 0.4.1b20220322, 0.4.1b20220323, 0.4.1b20220324, 0.4.1b20220325, 0.4.1b20220326, 0.4.1b20220327, 0.4.1b20220328, 0.4.1b20220329, 0.4.1b20220330, 0.4.1b20220331, 0.4.1b20220401, 0.4.1b20220402, 0.4.1b20220403, 0.4.1b20220404, 0.4.1b20220405, 0.4.1b20220406, 0.4.1b20220407, 0.4.1b20220408, 0.4.1b20220409, 0.4.1b20220410, 0.4.1b20220411, 0.4.1b20220412, 0.4.1b20220413, 0.4.1b20220414, 0.4.1b20220415, 0.4.1b20220416, 0.4.1b20220417, 0.4.1b20220418, 0.4.1b20220419, 0.4.1b20220420, 0.4.1b20220421, 0.4.1b20220422, 0.4.1b20220423, 0.4.1b20220424, 0.4.1b20220425, 0.4.1b20220426, 0.4.1b20220427, 0.4.1b20220428, 0.4.1b20220429, 0.4.1b20220430, 0.4.1b20220501, 0.4.1b20220502, 0.4.1b20220503, 0.4.1b20220504, 0.4.1b20220505, 0.4.1b20220506, 0.4.1b20220507, 0.4.1b20220508, 0.4.1b20220509, 0.4.1b20220510, 0.4.1b20220511, 0.4.1b20220512, 0.4.1b20220513, 0.4.1b20220514, 0.4.1b20220515, 0.4.1b20220516, 0.4.1b20220517, 0.4.1b20220518, 0.4.1b20220519, 0.4.1b20220520, 0.4.1, 0.4.2b20220521, 0.4.2b20220522, 0.4.2b20220523, 0.4.2b20220524, 0.4.2b20220525, 0.4.2b20220526, 0.4.2b20220527, 0.4.2b20220528, 0.4.2b20220529, 0.4.2b20220530, 0.4.2b20220531, 0.4.2, 0.4.3b20220601, 0.4.3b20220602, 0.4.3b20220603, 0.4.3b20220604, 0.4.3b20220605, 0.4.3b20220606, 0.4.3b20220607, 0.4.3b20220608, 0.4.3b20220609, 0.4.3b20220610, 0.4.3b20220611, 0.4.3b20220612, 0.4.3b20220613, 0.4.3b20220614, 0.4.3b20220615, 0.4.3b20220616, 0.4.3b20220617, 0.4.3b20220618, 0.4.3b20220619, 0.4.3b20220620, 0.4.3b20220621, 0.4.3b20220622, 0.4.3, 0.5.0b20220623, 0.5.0rc1, 0.5.0, 0.5.1b20220624, 0.5.1b20220625, 0.5.1b20220626, 0.5.1b20220627, 0.5.1b20220628, 0.5.1b20220629, 0.5.1b20220630, 0.5.1b20220701, 0.5.1b20220702, 0.5.1b20220703, 0.5.1b20220704, 0.5.1b20220705, 0.5.1b20220706, 0.5.1b20220707, 0.5.1b20220708, 0.5.1b20220709, 0.5.1b20220710, 0.5.1b20220711, 0.5.1b20220712, 0.5.1b20220713, 0.5.1b20220714, 0.5.1b20220715, 0.5.1b20220716, 0.5.1b20220717, 0.5.1b20220718, 0.5.1, 0.5.2b20220719, 0.5.2b20220720, 0.5.2b20220721, 0.5.2b20220722, 0.5.2b20220723, 0.5.2b20220724, 0.5.2b20220725, 0.5.2b20220726, 0.5.2b20220727, 0.5.2b20220728, 0.5.2, 0.5.3b20220729, 0.5.3b20220730, 0.5.3b20220731, 0.5.3b20220801, 0.5.3b20220802, 0.5.3b20220803, 0.5.3b20220804, 0.5.3b20220805, 0.5.3b20220806, 0.5.3b20220807, 0.5.3b20220808, 0.5.3b20220809, 0.5.3b20220810, 0.5.3b20220811, 0.5.3b20220812, 0.5.3b20220813, 0.5.3b20220814, 0.5.3b20220815, 0.5.3b20220816, 0.5.3b20220817, 0.5.3b20220818, 0.5.3b20220819, 0.5.3b20220820, 0.5.3b20220821, 0.5.3b20220822, 0.5.3b20220823, 0.5.3b20220824, 0.5.3b20220825, 0.5.3b20220826, 0.5.3b20220827, 0.5.3b20220828, 0.5.3b20220829, 0.5.3b20220830, 0.5.3b20220831, 0.5.3b20220901, 0.5.3b20220902, 0.5.3b20220903, 0.5.3b20220904, 0.5.3b20220905, 0.5.3b20220906, 0.5.3b20220907, 0.5.3b20220908, 0.5.3b20220909, 0.5.3b20220910, 0.5.3b20220911)
ERROR: No matching distribution found for autogluon.common==0.5.3b20220913

**Comment by Enderfga** [comment]
Unfortunately, no matter which version I install, I can't train properly with contains irregularly sampled time series, is there any other way to help me?

**Comment by shchur** [comment]
Please double check that the installed AG version is from the master branch (potentially installing from source), the output shown above indicates that the installation process failed. The error message should also look different in the new version. Also, it's hard to understand the cause of the problem without seeing the full stack trace.

**Comment by Enderfga** [comment]
I tried many times to install the latest version and each time it showed no frequency provided, my data is in hourly intervals but I really wanted to use the autogluon feature so I used a really stupid way to get the code to run, I temporarily modified ts_dataframe.py

![image](https://user-images.githubusercontent.com/74981769/189832132-ca909b73-f61c-4a79-a39b-cb1d988418e6.png)

**Comment by Enderfga** [comment]
Although the code is running, it is still very slow with such a large amount of data, is there any way to use GPU acceleration?

**Comment by shchur** [comment]
The model will automatically train on GPU if you installed a [GPU version of MXNet](https://mxnet.apache.org/versions/1.9.1/get_started?platform=linux&language=python&processor=gpu&environ=pip&).

**Comment by Innixma** [comment]
Marking as resolved. Please create a new issue if problem persists in latest AutoGluon release.
